doc/mainpage.dox

```diff
@@ -223,5 +223,106 @@ before the measurement framework or in a particular Plugin/Algorithm not knowing
 failure modes.  This means that Plugin/Algorithm implementations should generally have a global try/catch
 block that re-throwns lower-level exceptions corresponding to known failure modes as @ref MeasurementErrors.
 
+@section measBaseFlagHandlerInPython Using a FlagHandler with Python Plugins
+
+As mentioned above, a plugin measure() method is used to implement the code which performs
+measurements on individual detected sources.  It also records the results in the output
+source catalog. If an error occurs during measure(), the plugin is obliged to record the error by setting a
+flag in the source catalog.  As described above, this is often done by raising a MeasurementError and relying
+on the fail() method of the plugin to set the error flags.
```

Comment:
I do agree that the writing is unnecessarily verbose.  But I also think that some things which you think are unnecessary are needed to give the user context.  So I will try to clean up the writing, but still supply context.